core/src/main/java/bisq/core/btc/wallet/TradeWalletService.java

```diff
@@ -1183,11 +1183,8 @@ private RawTransactionInput getRawInputFromTransactionInput(@NotNull Transaction
                 "input.getConnectedOutput().getParentTransaction() must not be null");
         checkNotNull(input.getValue(), "input.getValue() must not be null");
 
-        // If we would be sure all bisq nodes migrated to bitcoinj 0.15, bitcoinSerialize(true) can be used.
-        // Since there is no way to check migration to bitcoinj 0.15,
-        // Config.SEGWIT_TRADE_PROTOCOL_ACTIVE could be checked instead.
-        // Actually, given the serialized tx is used just as the connected tx of an input,
-        // we don't need the connected tx's witness data.
+        // bitcoinSerialize(false) is used just in case the serialized tx is parsed by a bisq node still using
+        // bitcoinj 0.14. The serialized tx is just used to obtain its hash, so the witness data is not relevant.
         return new RawTransactionInput(input.getOutpoint().getIndex(),
```

Comment:
getRawInputFromTransactionInput is used only by methods for creating the deposit tx. With the trade protocol update we do not have mixed cases of segwit/legacy here. Or do I miss something?

Reply:
yes, you are right, just fixed the comment to explain that.